app/src/main/java/com/capstone/event_finder/activities/MainActivity.java

```diff
@@ -9,37 +9,51 @@
 import androidx.appcompat.app.AppCompatActivity;
 import androidx.fragment.app.Fragment;
 import androidx.fragment.app.FragmentManager;
+import androidx.lifecycle.ViewModelProvider;
 
 import com.capstone.event_finder.R;
 import com.capstone.event_finder.fragments.ExploreFragment;
 import com.capstone.event_finder.fragments.FeedFragment;
 import com.capstone.event_finder.fragments.ProfileFragment;
+import com.capstone.event_finder.interfaces.FeedFragmentInterface;
+import com.capstone.event_finder.network.EventViewModel;
 import com.google.android.material.bottomnavigation.BottomNavigationView;
 import com.parse.ParseUser;
 
 public class MainActivity extends AppCompatActivity {
+    private EventViewModel eventViewModel;
+    private FeedFragmentInterface listener;
```

Comment:
I had some trouble figuring out how to call getAPIEvents() from the fragment from the Main Activity and found that using an interface would allow me to do so - would love to discuss the other way I could have implemented this and when to use abstractions.